internal/ingress/controller/template/configmap.go

```diff
@@ -317,30 +319,41 @@ func ReadConfig(src map[string]string) config.Configuration {
 
 	// Nginx Status whitelist
 	if val, ok := conf[nginxStatusIpv4Whitelist]; ok {
-		whitelist := make([]string, 0)
-		whitelist = append(whitelist, strings.Split(val, ",")...)
-		to.NginxStatusIpv4Whitelist = whitelist
+		to.NginxStatusIpv4Whitelist = strings.Split(val, ",")
+		for i := range to.NginxStatusIpv4Whitelist {
+			to.NginxStatusIpv4Whitelist[i] = strings.TrimSpace(to.NginxStatusIpv4Whitelist[i])
+		}
 
 		delete(conf, nginxStatusIpv4Whitelist)
 	}
+
 	if val, ok := conf[nginxStatusIpv6Whitelist]; ok {
-		whitelist := make([]string, 0)
-		whitelist = append(whitelist, strings.Split(val, ",")...)
-		to.NginxStatusIpv6Whitelist = whitelist
+		to.NginxStatusIpv6Whitelist = strings.Split(val, ",")
+		for i := range to.NginxStatusIpv6Whitelist {
+			to.NginxStatusIpv6Whitelist[i] = strings.TrimSpace(to.NginxStatusIpv6Whitelist[i])
+		}
```

Comment:
These do not seem to be related to the PR - can you do them in a separate PR?

Reply:
done